src/test/scala/com/databricks/spark/sql/perf/mllib/MLLibTest.scala

```diff
@@ -0,0 +1,16 @@
+package com.databricks.spark.sql.perf.mllib
+
+import org.scalatest.FunSuite
+
+class MLLibTest extends FunSuite {
+
+  test("test MlLib benchmarks with mllib-small.yaml.") {
+    val results = MLLib.run(yamlConfig = MLLib.smallConfig)
+    val failures = results.na.drop(Seq("failure"))
+    if (failures.count > 0) {
+      val failed = failures.select("name").collect.map(_.getString(0)).mkString(", ")
```

Comment:
* ditto for `collect` -> `collect()`
* minor: this could be `failures.select("name").as[String].collect().mkString(", ")`

Reply:
I actually tried that, but I think I need to import `spark.implicits._` for that to work. Can I import those implicits from this context?

Reply:
You can do `import failures.sparkSession.implicits._`. Not necessary to change this PR though.